src/devices/Units/Temperature.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Units
```

Comment:
should the name space will have System? something like System.Iot.Unit? also did we use Iot casing same way in other places?

Reply:
Currently we use `Iot.*` everywhere. I wouldn't rename namespace or add a prefix in this PR - we can consider changing that after having more discussions later.

Reply:
please track it so will not get lost. having such current namespace looks wrong to me.

Reply:
https://github.com/dotnet/iot/issues/315